CHANGELOG

```diff
@@ -3,6 +3,21 @@ https://github.com/rtrlib/rtrlib/commits/master
 
 Release History:
 
+* Version 0.8.0
+    - fix blocking when stopping sockets in some cases
+    - improve error output in TCP transport
+    - properly initialise structures in rtrclient
+    - add callback to allow user created sockets
+    - Fix several compiler warnings
+    - Apply kernel coding style in all source files
+    - Add ability to modify the connect timeout
+    - Improve libssh detection
+    - Fix log prefixes in ssh transport
+    - Add support for password authentication to ssh transport
+    - Add support for password authentication for ssh transport to tools
+    - Fix constant spinning on receive with the ssh transport
```

Comment:
What does "constant spinning" mean?

Also: ... while receiving using ssh transport

Reply:
It constantly called the `tr_recv` function instead of blocking on it, because the implementation ignored the timeout argument and used a hard coded value of one second.

Reply:
Fix bug that constantly called the tr_recv because of ignoring timeout argument when using ssh transport.